templates/core/build-error.html

```diff
@@ -0,0 +1,15 @@
+{% extends "core/base_core.html" %}
+
+{% block title %}Build error | Ubuntu image builder{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1Hw_8dWAAUpHHm5Z_XNUVJaSVE5k5i2Vt6WANcJYUZvY/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row">
+    <h1>Build error</h1>
```

Comment:
Please wrap in another div with `col-8` to restrict the width of the error message.